charts/reform-scan-notification-service/values.aat.template.yaml

```diff
@@ -4,3 +4,19 @@ java:
   ingressHost: ${SERVICE_FQDN}
   environment:
     NOTIFICATIONS_CONSUME_TASK_DELAY_IN_MS: 1000
+    STAGING_QUEUE_NOTIFICATIONS_ENABLED: true
+    QUEUE_NOTIFICATIONS_READ: false
+  keyVaults:
+    reform-scan:
+      resourceGroup: reform-scan
+      secrets:
+        - app-insights-instrumentation-key
+        - error-notifications-password
+        - error-notifications-url
+        - error-notifications-username
+        - notification-service-POSTGRES-PASS
+        - notification-service-POSTGRES-USER
+        - notification-service-POSTGRES-HOST
+        - notification-service-POSTGRES-PORT
+        - notification-service-POSTGRES-DATABASE
+        - notifications-staging-queue-listen-connection-string
```

Comment:
you don't need to add all here it will use values from values yaml and merge aat template values

Reply:
we need to exclude 'notifications-queue-listen-connection-string'

Reply:
will it not merge values + aat template ? So you will get it anyways

Reply:
if you use secret it uses the one you use.